gcloud/monitoring/_dataframe.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Time series as `pandas`_ dataframes.
+
+.. _pandas: http://pandas.pydata.org/pandas-docs/stable/
+"""
+
+import itertools
+
+TOP_RESOURCE_LABELS = [
+    'project_id',
+    'aws_account',
+    'location',
+    'region',
+    'zone',
+]
+
+
+def _build_dataframe(time_series_iterable,
+                     label=None, labels=None):  # pragma: NO COVER
```

Comment:
There is no pandas in the `testenv:cover` environment, so the unit tests for this function are not executed.